cmd/kube-scheduler/app/server_test.go

```diff
@@ -154,6 +157,44 @@ profiles:
 		t.Fatal(err)
 	}
 
+	// out-of-tree plugin config v1beta3
+	outOfTreePluginConfigFilev1beta3 := filepath.Join(tmpDir, "outOfTreePluginv1beta3.yaml")
+	if err := os.WriteFile(outOfTreePluginConfigFilev1beta3, []byte(fmt.Sprintf(`
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+clientConnection:
+  kubeconfig: "%s"
+profiles:
+- plugins:
+    preFilter:
+      enabled:
+      - name: Foo
+    filter:
+      enabled:
+      - name: Foo
+`, configKubeconfig)), os.FileMode(0600)); err != nil {
+		t.Fatal(err)
+	}
+
+	// plugin config
```

Comment:
nit: replace with `out-of-tree plugin config v1beta2`

Reply:
will update along with other comments.